auraed/Cargo.toml

```diff
@@ -33,42 +33,34 @@ name = "auraed"
 version = "0.1.0"
 edition = "2021"
 authors = ["The Aurae Authors", "Kris Nóva <kris@nivenly.com>"]
+license = "Apache-2.0"
 
 [[bin]]
 name = "auraed"
 path = "src/bin/main.rs"
 
 [dependencies]
+anyhow = {workspace = true}
+aurae-proto = { workspace = true }
+cgroups-rs = { git = "https://github.com/kata-containers/cgroups-rs", branch = "main"}
 clap = { version = "3.1.20", features = ["derive"] }
 fancy-regex = "0.10.0"
+futures = "0.3.23"
+ipnetwork = "0.20.0"
+libc = "0.2"
 log = "0.4.17"
 multi_log = "0.1.2"
+netlink-packet-route = "0.13.0" # Used for netlink_packet_route::rtnl::address::nlas definition
+rtnetlink = "0.11.0"
 simplelog = "0.12.0"
 syslog = "6.0.1"
-tonic = { version = "0.8", features = ["tls"] }
-prost = "0.11"
-tokio-stream = { version = "0.1", features = ["net", "sync"] }
-tokio = { version = "1.0", features = ["macros", "fs", "rt-multi-thread"] }
-futures = "0.3.23"
-h2 = "0.3.13"
-rustls = "0.20.6"
-anyhow = "1.0.65"
-libc = "0.2"
-walkdir = "2"
-ipnetwork = "0.20.0"
-rtnetlink = "0.11.0"
-netlink-packet-route = "0.13.0" # Used for netlink_packet_route::rtnl::address::nlas definition
 thiserror = "1.0.37"
-libcontainer     = { git = "https://github.com/containers/youki.git", tag = "v0.0.2" }
-liboci-cli     = { git = "https://github.com/containers/youki.git", tag = "v0.0.2" }
-aurae-proto = { path = "../aurae-proto", version="0.1.0" }
-cgroups-rs =  { git = "https://github.com/kata-containers/cgroups-rs", branch = "main"}
+tokio = { workspace = true, features = ["macros", "fs", "rt-multi-thread"] }
+tokio-stream = { version = "0.1", features = ["net", "sync"] }
+tonic = { workspace = true, features = ["tls"] }
 validation = { path = "../crates/validation", features=["regex"] }
 validation_macros = { path = "../crates/validation/macros" }
-
-[build-dependencies]
-anyhow = "1.0.65"
-tonic-build = "0.8"
+walkdir = "2"
```

Comment:
This doesn't seem right. The compiler is probably allowing it because we are using it in what is temporarily dead code: https://github.com/aurae-runtime/aurae/blob/dfaacc1d5e583965aa527a192654de913a29c329/auraed/src/init/fileio.rs#L34

Reply:
i haven't removed this either.

Reply:
Nope. My mistake. Saw `[build-dependencies]` and thought you moved it there. It didn't click in my brain that that line is not going to be there.